source/pkcore/abilitypages/chi/highjump.rst

```diff
@@ -0,0 +1,60 @@
+.. highjump:
+###########
+HighJump
+###########
+
+Description
+###########
+
+HighJump is a Chi Ability.
+
+HighJump allows the Chiblocker to launch themselves high in the air, letting them dodge and surpass obstacles.
```

Comment:
Change to `HighJump allows Chiblockers to launch themselves high in the air, letting them dodge oncoming attacks and bypass obstacles. GIF TO BE ADDED`